oioioi/notifications/processors.py

```diff
@@ -20,6 +21,7 @@ def generate_token():
     return new_token
 
 
+@request_cached
 def get_notifications_session(session):
```

Comment:
the argument of this function is a session, not a request - are you sure `@request_cached` decorator will work correctly on it?

Reply:
Indeed it is in violation with request_cached docstring. To be honest I saw a commit adding a bunch of `request_cached` docorators and knowing it was operational on production made no further digging.

I think it would be best to remove this change here, and bring later it back provided it is safe and beneficial.

Reply:
yeah, the rest of the commit looks fine but this one line looks like it may've been mis-rebased due to a refactor, better skip it for now